AdyenActions/Components/3DS2/Action handlers/ThreeDS2ClassicActionHandler+Initializers.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import Adyen3DS2
+import Foundation
+#if canImport(AdyenAuthentication)
+    import AdyenAuthentication
+#endif
+
+extension ThreeDS2ClassicActionHandler {
+    
+    /// Initializes the 3D Secure 2 action handler.
+    internal convenience init(context: AdyenContext,
+                              appearanceConfiguration: ADYAppearanceConfiguration,
+                              delegatedAuthenticationConfiguration: ThreeDS2Component.Configuration.DelegatedAuthentication?) {
+        self.init(
+            context: context,
+            appearanceConfiguration: appearanceConfiguration,
+            coreActionHandler: createDefaultThreeDS2CoreActionHandler(
+                context: context,
+                appearanceConfiguration: appearanceConfiguration,
+                delegatedAuthenticationConfiguration: delegatedAuthenticationConfiguration
+            ),
+            delegatedAuthenticationConfiguration: delegatedAuthenticationConfiguration
+        )
```

Comment:
```suggestion
        let defaultHandler = createDefaultThreeDS2CoreActionHandler(
            context: context,
            appearanceConfiguration: appearanceConfiguration,
            delegatedAuthenticationConfiguration: delegatedAuthenticationConfiguration
        )
        self.init(
            context: context,
            appearanceConfiguration: appearanceConfiguration,
            coreActionHandler: defaultHandler,
            delegatedAuthenticationConfiguration: delegatedAuthenticationConfiguration
        )
```

How about moving result of `createDefaultThreeDS2CoreActionHandler` into local variable? Calling method with 4 params inside other method with 4 params is a bit heavy syntax IMHO